### PR TITLE
BROWSER-95: Refactor cliqz.cfg

### PR DESCRIPTION
**Pull request by maxim-cliqz**

### CODE REVIEW

File: mozilla-release/browser/confvars.sh [hunk 1]
```diff
@@ -35,8 +35,8 @@ MOZ_SERVICES_COMMON=1
 MOZ_SERVICES_CRYPTO=1
 MOZ_SERVICES_HEALTHREPORT=1
 MOZ_SERVICES_METRICS=1
-MOZ_SERVICES_SYNC=1
```
Comment (chrmod) [inline mozilla-release/browser/confvars.sh]: maybe we can find a better way to alter those vars. One that will not require changes in source files

**Comment by chrmod** [comment]
retest this please

File: mozilla-release/browser/confvars.sh (hunk 1 above)
Comment (maxim-cliqz) [inline mozilla-release/browser/confvars.sh]: BTW, we may also want to disable mozilla health report service (see line 36). @luciancor ?

Comment (alver-cliqz) [inline mozilla-release/browser/confvars.sh]: Better to redirect this to our URL and put a dummy handler for it. Never knows when it could be useful for us

**Pull request merged**